sormas-ui/src/main/java/de/symeda/sormas/ui/symptoms/SymptomsForm.java

```diff
@@ -76,623 +77,684 @@
+                            fluidColumn(6, 0,
+                                    locsCss(VSPACE_3,
+                                            PALPABLE_LIVER, PALPABLE_SPLEEN, PHARYNGEAL_ERYTHEMA, PHARYNGEAL_EXUDATE,
+                                            PIGMENTARY_RETINOPATHY, PNEUMONIA_CLINICAL_OR_RADIOLOGIC,
+                                            PURPURIC_RASH, RADIOLUCENT_BONE_DISEASE, RAPID_BREATHING,
+                                            REFUSAL_FEEDOR_DRINK, RUNNY_NOSE,
+                                            ORAL_ULCERS, SIDE_PAIN, SORE_THROAT, SPLENOMEGALY, SUNKEN_EYES_FONTANELLE
+                                            , SWOLLEN_GLANDS,
+                                            THROBOCYTOPENIA, TREMOR, UNEXPLAINED_BLEEDING, EYES_BLEEDING,
+                                            INJECTION_SITE_BLEEDING,
+                                            BLEEDING_VAGINA, GUMS_BLEEDING, STOMACH_BLEEDING, BLOOD_URINE,
+                                            BLOODY_BLACK_STOOL,
+                                            SKIN_BRUISING, COUGHING_BLOOD, DIGESTED_BLOOD_VOMIT, RED_BLOOD_VOMIT,
+                                            NOSE_BLEEDING,
+                                            OTHER_HEMORRHAGIC_SYMPTOMS, OTHER_HEMORRHAGIC_SYMPTOMS_TEXT,
+                                            LESIONS, LESIONS_THAT_ITCH, LESIONS_SAME_STATE, LESIONS_SAME_SIZE,
+                                            LESIONS_DEEP_PROFOUND,
+                                            LESIONS_LOCATIONS_LOC, LESIONS_FACE, LESIONS_LEGS, LESIONS_SOLES_FEET,
+                                            LESIONS_PALMS_HANDS, LESIONS_THORAX,
+                                            LESIONS_ARMS, LESIONS_GENITALS, LESIONS_ALL_OVER_BODY,
+                                            LESIONS_RESEMBLE_IMG1, MONKEYPOX_LESIONS_IMG1,
+                                            LESIONS_RESEMBLE_IMG2, MONKEYPOX_LESIONS_IMG2, LESIONS_RESEMBLE_IMG3,
+                                            MONKEYPOX_LESIONS_IMG3, LESIONS_RESEMBLE_IMG4, MONKEYPOX_LESIONS_IMG4,
+                                            LESIONS_ONSET_DATE, VOMITING, HYDROPHOBIA, OPISTHOTONUS, HYPERACTIVITY,
+                                            PARESIS, AGITATION,
+                                            ASCENDING_FLACCID_PARALYSIS, ERRATIC_BEHAVIOUR, COMA, LOSS_OF_TASTE,
+                                            LOSS_OF_SMELL, WHEEZING, SKIN_ULCERS, INABILITY_TO_WALK,
+                                            IN_DRAWING_OF_CHEST_WALL,
+                                            OTHER_NON_HEMORRHAGIC_SYMPTOMS, OTHER_NON_HEMORRHAGIC_SYMPTOMS_TEXT) +
+                                            locsCss(VSPACE_3, PATIENT_ILL_LOCATION, SYMPTOMS_COMMENTS)
+                            )
+                    ) +
+                    fluidRowLocsCss(VSPACE_3, ONSET_SYMPTOM, ONSET_DATE) +
+                    loc(COMPLICATIONS_HEADING) +
```

Comment:
Please find a way to group all "symptoms" that belong to the complications section in the SymptomsDto so that it's easier for developers to see the difference between normal symptoms and complcations.
Then enable the new "other complications" for all diseases that use at least one of those complications.